package.json

```diff
@@ -9,13 +9,14 @@
     "preview": "vite preview",
     "prepare": "husky",
     "test": "vitest",
-    "test:dev": "vitest watch"
+    "test:coverage": "vitest --coverage"
```

Comment:
¿Por qué quitáis el script `test:dev`?

Reply:
Porque el el test de vitest ya lo ejecuta en modo watch

Reply:
Pero necesitas un script que los tire en modo watch y otro que los tire una vez.

Reply:
Ya lo hemos añadido:
    "test": "vitest run",
    "test:dev": "vitest",